examples/perplexity/perplexity.cpp

```diff
@@ -813,7 +813,6 @@ static void hellaswag_score(llama_context * ctx, const gpt_params & params) {
 
     llama_batch batch = llama_batch_init(n_ctx, 0, max_seq);
 
-    std::vector<float> tok_logits(n_vocab);
```

Comment:
This is not unused.

Reply:
Yup, got a bit carried away with the static analyzer, reverted this